ci/gitlab/tests/pr_open_01.json

```diff
@@ -11,7 +11,8 @@
         "source": {
             "http_url": "https://gitlab.com/testmb01/test_repo.git",
             "visibility_level": 20,
-            "namespace": "testmb01",
+            "namespace": "Alger, Brian",
+            "path_with_namespace": "testmb01/test_repo",
```

Comment:
@brianmoose sorry to tag you... 'namespace' is what changed in GitLab's API, prompting this PR. Namespace now returns your full name with special characters, spaces etc etc. Worse yet, [according to GitLab](https://docs.gitlab.com/ee/api/README.html#namespaced-path-encoding), you can use namespace with url encoding to access the same resources as before. But no. [There is an open issue](https://gitlab.com/gitlab-org/gitlab-foss/issues/35079) claiming this is broken (possibly by nginx mangling the encoding). Indeed, we are hitting this issue.

Love their API.

Anyways, are you comfortable with your name here? This is the information which _would_ return in the webhook payload. But I doubt that we are actually calling 'https://gitlab.com/testmb01/test_repo.git' during testing. ...so we can change this to what ever we want... yes?

Reply:
Ugh, I always hated their API.
We do not actually use that repo anywhere. I just had a script that would generate those JSON payloads against that repo so that we would have something realistic to test against. The values just have to match what is used in the tests.
Using my name is fine but you can put whatever you want in there..... I think :)